src/server/GraphQLEngine/graphQLSchema/types/PodList.ts

```diff
@@ -18,11 +18,12 @@ export default () => [
   type PodList {
     kind: String!
     apiVersion: String!
-    items: [PodElement]!
+    items: [PodElement]
```

Comment:
Maybe the optional fields are the array elements not array itself
```suggestion
    items: [PodElement]!
```